examples/capi-quick-start/nutanix-cluster-calico-helm-addon.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Please remove `examples/capi-quick-start/aws-cluster-identity.yaml` and `examples/capi-quick-start/nutanix-cluster-calico-helm-addon.yaml`.

Reply:
removed aws-cluster-identity.yaml but nutanix-cluster-calico-helm-addon.yaml is now up to date